src/wallet/wallet.h

```diff
@@ -88,6 +88,10 @@ enum WalletFeature
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
     FEATURE_HD = 130000, // Hierarchical key derivation after BIP32 (HD Wallet)
+
+    //TODO: FEATURE_HD_SPLIT needs to be bumped to 140000 once we branch of 0.14 //
```

Comment:
It should not be required to bump it when it is in master, because some people are using master "in production" (i.e. on the main net).